src/routers/handlers/email/changeEmailAddress.ts

```diff
@@ -104,16 +69,3 @@ export class ChangeEmailAddressHandler extends GenericHandler {
     return Promise.resolve(this.viewData);
   }
 }
-
-// create transaction record
-export const createTransaction = async (session: Session, companyNumber: string): Promise<string> => {
```

Comment:
you've removed the `createTransaction` async handler - so there's no transaction record being created on successful authentication for a company? Where's the transaction record being created?

Reply:
Yep this got removed by mistake! A bit trigger happy after removing the stuff above and explains why it can't close a transaction at the end!